src/compiler/crystal/syntax/parser.cr

```diff
@@ -4709,7 +4709,7 @@ module Crystal
       intel = false
       while true
         location = @token.location
-        option = parse_string_without_interpolation("asm clobber")
+        option = parse_string_without_interpolation("asm option")
```

Comment:
Which sounds better error message: `interpolation not allowed in asm option` or `interpolation not allowed in asm options`? I'm not native speaker, so I don't know.